routes/accounts.js

```diff
@@ -19,12 +19,15 @@ router.get( '/register', ( request, response ) => {
 
 router.post( '/register', ( request, response , next) => {
   const User = request.app.get( 'models' ).User
+  const Item = request.app.get( 'models' ).Item
 
   const { email, password } = request.body
 
   User.create({ email, password: encryptPassword( password ) })
+    .then( user => Item.create({ is_root: true, parent_id: 0, title: 'Home', description: 'Welcome to Floworky', user_id: user.id }))
     .then( user => RegistrationEmail.send( user ))
     .then( user => response.redirect( '/accounts/verify' ))
+    .catch( error => response.render( 'accounts/verify' ))
```

Comment:
I think you need a forward slash before `accounts/verify`?  If you don't, would still prefer it so that it is consistent with the line above.